build.gradle

```diff
@@ -21,6 +21,13 @@ plugins {
     id 'com.bmuschko.docker-remote-api'             version '9.4.0'
     id 'io.github.gradle-nexus.publish-plugin'      version '2.0.0-rc-1'
     id 'org.gradle.crypto.checksum'                 version '1.4.0'
+    id "idea"
+}
+
+idea {
+    module {
+        downloadSources = true
```

Comment:
Just a little change to make working with Idea less painful. The plugin interfaces with Idea and causes the Gradle sync inside Idea to download library sources. 

I think that it will not affect jars and builds outside Idea (I checked that locally), but probably someone else should check on their side, on different configuration, too.